.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
- [ ] The Review suppose to contain at least 2 inline comments, kindly look for any mistakes or even 
 enhancements in your project and the Please write at least 2 comments in the review 💪🏼 

Refer to this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) for more information about writing inline comments, it's the last section of the article.

Reply:
Ok, @adel-gu appreciate your feedback, I will make the comment correctly.